pjsip/src/pjsip/sip_parser.c

```diff
@@ -1916,8 +1916,8 @@ static void int_parse_contact_param( pjsip_contact_hdr *hdr,
             hdr->expires = pj_strtoul(&pvalue);
             if (hdr->expires == PJSIP_EXPIRES_NOT_SPECIFIED)
                 hdr->expires--;
-            if (hdr->expires > PJSIP_MAX_EXPIRES)
-                hdr->expires = PJSIP_MAX_EXPIRES;
+            //if (hdr->expires > PJSIP_MAX_EXPIRES)
+            //    hdr->expires = PJSIP_MAX_EXPIRES;
```

Comment:
Perhaps also remove `MAX_EXPIRES` constant as it seems to be only used here? The RFC does not seem to specify the maximum expire time either.